### PR TITLE
heroku and AWS setup

### PR DESCRIPTION
**Pull request by Andrii-Kozak**
[Trello Task](https://trello.com/_____)

**CHECK LIST**
- [x] specs added
- [x] specs passed
- [x] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [ ] PR is reviewed manually again (to make sure you have 100% ready code)
- [ ] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs
- [ ] screenshot of feature is added if task has UI

https://calm-harbor-98985.herokuapp.com/

### CODE REVIEW

File: bin/spring [hunk 1]
```diff
@@ -1,17 +0,0 @@
-#!/usr/bin/env ruby
```
Comment (pereslop) [inline bin/spring:1]: why you removed this file?

**Review by marikrebega — APPROVED** [review]
but return spring file

File: bin/spring (hunk 1 above)
Comment (Andrii-Kozak) [reply bin/spring:1]: because for heroku there was an old version

**Review by IvanDanyliv — APPROVED** [review]

File: bin/bundle [hunk 2]
```diff
@@ -38,8 +38,8 @@ m = Module.new do
   end
 
   def gemfile
-    gemfile = ENV["BUNDLE_GEMFILE"]
-    return gemfile if gemfile.present?
+     gemfile = ENV["BUNDLE_GEMFILE"]
```
Comment (pereslop) [inline bin/bundle:41]: additional space here

File: bin/bundle [hunk 3]
```diff
@@ -38,8 +38,8 @@ m = Module.new do
...
+     return gemfile unless gemfile.nil? || gemfile.empty?
```
Comment (pereslop) [inline bin/bundle:42]: and here

File: config/environments/production.rb [hunk 4]
```diff
@@ -116,7 +116,7 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.raise_delivery_errors = true
   config.action_mailer.perform_caching = false
-  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.default_url_options = { host: 'calm-harbor-98985.herokuapp.com/' }
```
Comment (pereslop) [inline config/environments/production.rb:119]: will it work this URL work in development  mode?

File: bin/bundle (hunk 3 above)
Comment (Andrii-Kozak) [reply bin/bundle:42]: done)

**Pull request merged**